aries-core/src/js/components/helpers/Anchors/AnchorNavLink.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Anchor, ThemeContext } from 'grommet';
+import { css } from 'styled-components';
+
+export const AnchorNavLink = ({ ...rest }) => {
+  return (
+    <ThemeContext.Extend
+      value={{
+        anchor: {
+          extend: css`
+            opacity: 70%;
+            ${props => props.active && 'opacity: 100;'}
+          `,
+          hover: {
+            extend: {
+              opacity: '100%',
+            },
+            textDecoration: 'none',
+          },
+          textDecoration: 'none',
+        },
+      }}
```

Comment:
Instead of applying ThemContext every time we use Anchor, maybe it will be a better practice to apply those styling on the theme itself, and remove all usage of ThemeContext

Reply:
I think that I am in agreement. It was part of what I was touching on yesterday when mentioning my feeling that the aries theme belongs with aries-core rather than with aries-site. Right now core is consuming grommet-theme-hpe, which is why these helper components are applying ThemeContext. If aries-core consumes the aries theme, then the specification can be done on the theme file rather than in ThemeContext.

Reply:
The theme can be a custom theme part of aries-site.
I think aries-core should be able the render its components no matter what theme it will be given, and besides the HPE theme it shouldn't have a custom theme by itself. Regardless, this styling behavior is specific for aries-site so it should be placed on its theme.